app/views/changesets/index.html.erb

```diff
@@ -4,7 +4,7 @@
   </ol>
 <% if @changesets.size == 20 -%>
   <div class="changeset_more mt-3 text-center">
-    <%= link_to t(".load_more"), url_for(@params.merge(:max_id => @changesets.last.id - 1)), :class => "btn btn-primary" %>
+    <%= link_to t(".load_more"), "#{url_for}/#{@changesets.last.id - 1}", :class => "btn btn-primary" %>
```

Comment:
That way of constructing the URL is not a good idea - doesn't `url_for(:max_id => @changesets.last.id - 1)` work? If not then there will something which does and which doesn't rely on manually gluing the component together like that.